types/src/on_chain_config/execution_config.rs

```diff
@@ -11,13 +11,17 @@ pub enum OnChainExecutionConfig {
     V1(ExecutionConfigV1),
     V2(ExecutionConfigV2),
     V3(ExecutionConfigV3),
+    /// To maintain backwards compatibility on replay, we must ensure that any new features resolve
+    /// to previous behavior (before OnChainExecutionConfig was registered) in case of Missing.
+    Missing,
```

Comment:
It is pretty easy to make a mistake and add the next version `V4` after `V3` and before `Missing` and thus breaking backward compatibility. Can you add a comment here to add V4 after `Missing` ?